docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -96,13 +99,29 @@ Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
 ### claimOwnership
 
 ```solidity
-function claimOwnership() public {
+function claimOwnership() public
 ```
 
 Transfers ownership of the contract to the `pendingOwner` address. Can only be called by the `pendingOwner`.
 
 _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the new owner has claimed ownership._
 
+### renounceOwnership
+
+```solidity
+function renounceOwnership() public
+```
+
+Renounces ownership of the contract in 2 steps. Can only be called by the owner.
```

Comment:
This is a v**ery sensitive function**. It should be emphasized much more in the docs what is really happening under the hood. Especially the fact that once `renounceOwnership(...)` is called:

- the contract is left without an owner (= the owner is `address(0)`).
- the functions marked as `onlyOwner` cannot be called anymore.

As a good example, the contract API docs of `renounceOwnership` from OpenZeppelin is very cleat about this: https://docs.openzeppelin.com/contracts/4.x/api/access#Ownable-renounceOwnership--